packages/eslint-config/index.js

```diff
@@ -1,24 +1,29 @@
 module.exports = {
   extends: [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react/recommended",
-    "turbo",
-    "prettier",
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
```

Comment:
Not sure why this package wasn't prettier'd before, but it looks like my IDE picked up the prettier config and formatted this. Adding this line was the only change.